pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -2027,6 +2027,11 @@ remoteAddress, new String(commandGetSchema.getSchemaVersion()),
         long requestId = commandGetSchema.getRequestId();
         SchemaVersion schemaVersion = SchemaVersion.Latest;
         if (commandGetSchema.hasSchemaVersion()) {
+            if (commandGetSchema.getSchemaVersion().length == 0) {
+                commandSender.sendGetSchemaErrorResponse(requestId, ServerError.IncompatibleSchema,
+                        "Empty schema version");
+                return;
+            }
```

Comment:
When I debugged `testAvroSchemaWithTcpLookup` and `testAvroSchemaWithHttpLookup`, it never went here. Then I removed these lines, tests still passed.